### DISCUSSION TITLE
反馈一个BUG

### DISCUSSION THREAD

**Discussion by daxiangzi166** [amll-dev/applemusic-like-lyrics#222] [General]
在版本号为Build(202358)3.0.0(Beta)的网易云在使用私人漫游时，直接点击私人漫游进入官方的播放器界面，而从下方播放栏进入的是插件的播放界面，插件版本为3.0.3(DEV-6839f200)
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/105540716/efc94028-98b9-4c6d-a1b5-443e023ca6e5)
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/105540716/2472dc30-33d7-471e-9cc0-04fb77e0aa69)
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/105540716/1e47ef3c-0ced-4489-82b4-63894f0a18e3)
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/105540716/92196e25-53cb-4c6c-9277-e20ca8ec06d9)

**Comment by Linho1219** [comment]
网易云音乐 3.x 于 AMLL bncm 插件存在冲突，如却有需要，可以使用 downgradeOK 插件降级前端至 2.x 版本后使用。当然，我们更推荐使用 WSConnector 插件配合 AMLL Player 展示歌词。

AMLL bncm 插件已停止维护，最终版本存档于 <https://github.com/amll-dev/applemusic-like-lyrics-proto>。现关闭此讨论，敬请谅解。
